Header hdr
time stamp
int32 control_mode
float32[12] Tend_desired
float32[12] Tend_actual
